src/app/clients/clients-routing.module.ts

```diff
@@ -149,7 +151,15 @@ const routes: Routes = [
         ]
       }
     ]
-  }
+  },
+  {
+    path: 'clients/:clientId/chargeoverview',
+    component: ChargesOverviewComponent,
+    data: { title: extract('Client Charge Overview'), breadcrumb: 'Client Charge Overview', routeParamBreadcrumb: false },
+    resolve: {
+      clientChargesData: ClientChargeOverviewResolver,
+    }
+  },
   ])
```

Comment:
routing is a problem here as well

Reply:
nest the clients and client id here itslef and add route param breadcrumb, breadcrumb should be `home / clients / 1 / charges overview

Reply:
make the same change for previous PR approved PR as well

Reply:
I'll refactor it

Reply:
Nested routings added. Please review